python-sdk/src/transfers/data_providers/aws/s3.py

```diff
@@ -0,0 +1,144 @@
+from __future__ import annotations
+
+import logging
+import os
+
+from airflow.hooks.dbapi import DbApiHook
+from airflow.providers.amazon.aws.hooks.s3 import S3Hook
+from transfers.constants import FileLocation, TransferMode
+from transfers.data_providers import create_dataprovider
+from transfers.data_providers.base import DataProviders
+from transfers.datasets.base import UniversalDataset as Dataset
+
+from astro.constants import LoadExistStrategy
```

Comment:
IF the plan is to separate this into its own package it will be worth seeing how much/how little we actually import from other astro SDK modules.

Reply:
(if this is separated I imagine astro SDK would import this rather than the other way around)